### ISSUE TITLE
Support for logical replication / change data capture (CDC)?

### ISSUE THREAD

**Issue by fritzblue** [edgedb/edgedb#3916]
The following workflow is becoming more and more common in modern applications:

![image](https://user-images.githubusercontent.com/6473463/170763327-84bdd54d-717d-4534-a56d-6268c224e3df.png)

In order to use edgedb as a primary data store in a production system, it needs to support logical replication / change data capture in some form (propagating changes into kafka or similar).

Is first class support for CDC planned? Or are users recommended to perform CDC [against postgres directly](https://debezium.io/documentation/reference/stable/connectors/postgresql.html)?

**Comment by elprans** [comment]
Right now running CDC against the underlying Postgres is the only way.  First-party support for EdgeDB-flavored CDC is planned.

**Comment by aretrace** [comment]
Any updates on this?

**Comment by crashinc99** [comment]
Yes, please, any update?   Also, we built a custom replication system that uses Debezium to perform CDC on Postgres.  We move the changes through a specialized transport system to a destination Postgres instance.  We convert the Debezium XML messages to SQL and apply to the destination Postgres system.   The specialized transport system is required.   Would this workflow break/interfere with EdgeDB?

**Comment by rem-aster** [comment]
Any updates?
